openedx_lti_tool_plugin/templates/openedx_lti_tool_plugin/courseware.html

```diff
@@ -0,0 +1,37 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset='utf-8'>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+  </head>
+
+  <body>
+    <iframe
+      id='unit-iframe'
+      src='{{ xblock_url }}'
+      allow='microphone *; camera *; midi *; geolocation *; encrypted-media *'
+      allowfullscreen=''
```

Comment:
This content is intended to be embedded on other platforms, so I don't think it is necessary to allow this attribute. @kuipumu